openssl/src/nid.rs

```diff
@@ -104,7 +104,7 @@ pub enum Nid {
     G,
     S,
     I,
-    UID,
+    uid, // uniqueIdentifier
```

Comment:
> In the mean time, let's add this as `UserId`.

Done.

> these variants should definitely be renamed 

It's more than renaming. There are missing variants, and the values have been shown in the past to be incorrect.